recipes/r-sandpaper/meta.yaml

```
@@ -0,0 +1,124 @@
+{% set version = '0.16.11' %}
+{% set posix = 'm2-' if win else '' %}
+
+package:
+  name: r-sandpaper
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://github.com/carpentries/sandpaper/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 2036381b8421f2e47afd8cfc6be56fa0aab70877da7bd773476a431a3f7b2517
+  patches:
+    # c.f. https://github.com/carpentries/sandpaper/pull/651
+    - add-lesson-example-to-config-template-source.patch
+
+build:
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'        # [win]
+    - '*/Rblas.dll'    # [win]
+    - '*/Rlapack.dll'  # [win]
+
+requirements:
+  build:
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}git
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-assertthat
+    - r-callr
+    - r-cli >=3.4.0
+    - r-commonmark
+    - r-desc
+    - r-fs >=1.5.0
+    - r-gert >=1.0.1
+    - r-gh
+    - r-glue
+    - r-knitr >=1.33
+    - r-pegboard >=0.7.0
+    - r-pkgdown >=1.6.0
+    - r-renv >=0.14.0
+    - r-rlang >=0.4.3
+    - r-rmarkdown >=2.4
+    - r-rprojroot
+    - r-rstudioapi
+    - r-servr
+    - r-usethis >=2.0.0
+    - r-whisker
+    - r-withr
+    - r-yaml
+    - r-r.utils
+  run:
+    - r-base
+    - r-assertthat
+    - r-callr
+    - r-cli >=3.4.0
+    - r-commonmark
+    - r-desc
+    - r-fs >=1.5.0
+    - r-gert >=1.0.1
+    - r-gh
+    - r-glue
+    - r-knitr >=1.33
+    - r-pegboard >=0.7.0
+    - r-pkgdown >=1.6.0
+    - r-renv >=0.14.0
+    - r-rlang >=0.4.3
+    - r-rmarkdown >=2.4
+    - r-rprojroot
+    - r-rstudioapi
+    - r-servr
+    - r-usethis >=2.0.0
+    - r-whisker
+    - r-withr
+    - r-yaml
+    - r-r.utils
+
+test:
+  requires:
+    - git
+    - r-testthat
+    - r-mockr
+    - r-sessioninfo
+    - r-varnish
+  source_files:
+    - tests/
+  commands:
+    - $R -e "library('sandpaper')"  # [not win]
+    - $R -e "testthat::test_file('tests/testthat.R', stop_on_failure=TRUE)"  # [not win]
+    - "\"%R%\" -e \"library('sandpaper')\""  # [win]
+    - "\"%R%\" -e \"testthat::test_file('tests/testthat.R', stop_on_failure=TRUE)\""  # [win]
```

Suggestion for debugging win fail:

```suggestion
test:
  requires:
    - git            # [not win]
    - r-testthat     # [not win]
    - r-mockr        # [not win]
    - r-sessioninfo  # [not win]
    - r-varnish      # [not win]
  source_files:
    - tests/
  commands:
    - $R -e "library('sandpaper')"  # [not win]
    - $R -e "testthat::test_file('tests/testthat.R', stop_on_failure=TRUE)"  # [not win]
    - "\"%R%\" -e \"library('sandpaper')\""  # [win]
    #- "\"%R%\" -e \"testthat::test_file('tests/testthat.R', stop_on_failure=TRUE)\""  # [win]
```